packages/snaps-controllers/src/interface/utils.ts

```diff
@@ -98,11 +106,16 @@ function constructInputState(
  */
 function constructFormInputState(
   oldState: InterfaceState,
-  component: InputElement | DropdownElement,
+  component: InputElement | DropdownElement | FileInputElement,
   form: string,
 ) {
   const oldFormState = oldState[form] as FormState;
   const oldInputState = oldFormState?.[component.props.name];
+
+  if (component.type === 'FileInput') {
```

Comment:
Curious if we could just have it fall into the return below instead?

Reply:
We can't. `FileInput` doesn't have a `value` prop.

Reply:
So it would be undefined and we would move to the next default? 🤔

Reply:
> TS2339: Property value does not exist on type InputProps | DropdownProps | FileInputProps

Reply:
Eh, that seems kinda dumb 🙃

Reply:
Could we cast? I'm just worried the complexity will pile up in this function if we add ifs for everything 😓

Reply:
We could but I rather don't. We can optimise if it becomes an issue in the future?

Reply:
Fair 👍